src/realm/sync/noinst/client_impl_base.cpp

```diff
@@ -1906,6 +1906,8 @@ void Session::send_bind_message()
         auto schema_version = get_schema_version();
         // Send 0 if schema is not versioned.
         bind_json_data["schemaVersion"] = schema_version != uint64_t(-1) ? schema_version : 0;
+        // Send 0 if sync client is not using a relaxed schema
+        bind_json_data["relaxedSchema"] = get_db()->flexible_schema_allowed() ? 1 : 0;
```

Comment:
Thoughts on adding a test that tries to get back one of the new error codes from the server to make sure the server is properly decoding this into a bool (rather than just always defaulting to false or something)? 

I don't think this should be an issue, but seems like it would be valuable to at the very least do a manual check

Reply:
Added a test to validate the "app doesn't support relaxed schema" case - turns out the server doesn't support an int for this value, so it was changed to a bool value.

I recommend updating this to be an int value in case we want to expand the meaning of this value in the future.